src/pages/home/Home.jsx

```diff
@@ -1,7 +1,179 @@
+					collaborate with.
+				</p>
+				<div className="py-4 ">
+					<button className="px-5 py-3 bg-primary-color rounded-md flex flex-row items-center gap-3 hover:opacity-75">
+						<p className="text-primary-bg-color text-lg font-bold ">
+							Join Peerlist
+						</p>
+						<div>
+							<svg
+								xmlns="http://www.w3.org/2000/svg"
+								className="h-5 w-5 fill-white"
+								viewBox="0 0 20 20"
+							>
+								<path
+									fillRule="evenodd"
+									d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-8.707l-3-3a1 1 0 00-1.414 1.414L10.586 9H7a1 1 0 100 2h3.586l-1.293 1.293a1 1 0 101.414 1.414l3-3a1 1 0 000-1.414z"
+									clipRule="evenodd"
+									className="bg-primary-bg-color"
+								/>
+							</svg>
+						</div>
+					</button>
+				</div>
+			</div>
+			<div className=" bg-bg-off-white flex sm:flex-row flex-col items-center justify-center md:px-20 p-5">
+				<div className="flex-1  flex flex-col gap-5">
+					<p className="text-lg text-primary-font-color">Peerlist scroll</p>
+					<p className="md:text-6xl sm:text-5xl text-4xl leading-tight font-extrabold text-secondary-bg-color ">
+						A feed of opportunities!
+					</p>
+					<p className="text-lg">
+						Peerlist Scroll is designed for the distribution and discovery of
+						your work. But, you never know, that new side-project you shared
+						might land you an opportunity of a lifetime.
+					</p>
+					<p className="text-lg">Finite scroll with infinite possibilities ∞</p>
+					<div className="grid sm:grid-cols-2 grid-cols-1 gap-2 items-center justify-center">
+						{opportunityData.map((item, index) => {
+							return (
+								<HomeCard
+									key={index}
+									image_src={item.image_src}
+									title={item.title}
+									description={item.description}
+								/>
+							);
+						})}
+					</div>
+				</div>
+				<img
+					src={illustration_scroll}
+					alt="illustration-scroll"
+					className="flex-1"
+				/>
+			</div>
+			<div className=" bg-primary-bg-color flex sm:flex-row flex-col items-center justify-center md:px-20 p-5">
+				<div className="flex-1 bg-primary-bg-color flex flex-col gap-5">
+					<p className="text-lg text-primary-font-color">Networking</p>
+					<p className="md:text-6xl sm:text-5xl text-4xl leading-tight font-extrabold text-secondary-bg-color ">
+						Become Peers, connections
+					</p>
+					<p className="text-lg">
+						Develop meaningful relationships with talented peers worldwide that
+						will last far beyond your traditional connection requests.
+					</p>
+					<p className="text-lg">
+						PS: You “connect” only when you follow each other !.
+					</p>
+					<div className="grid sm:grid-cols-2 grid-cols-1 gap-2 items-center justify-center">
+						{connectionData.map((item, index) => {
+							return (
+								<HomeCard
+									key={index}
+									image_src={item.image_src}
+									title={item.title}
+									description={item.description}
+								/>
```

Comment:
suggestion: can send the item itself and destructure in HomeCard.jsx

Reply:
implmented suggesion